buildsettings.json

```diff
@@ -5,7 +5,7 @@
         "minVersion": "20.1.3"
     },
     "kubernetes": {
-        "maxVersion": "1.21",
-        "minVersion": "1.16"
+        "maxVersion": "1.22",
+        "minVersion": "1.19"
```

Comment:
We are reducing the support matrix to 3 releases only from an earlier 5 releases.

Reply:
*6 to 4 releases yea. We are essentially deprecating 1.16,1.17,1.18 and adding support for 1.21. I think we were fine with this though. cc: @apalsule